chromatic/rainbows/actions/flag_outliers.py

```diff
@@ -0,0 +1,50 @@
+from ...imports import *
+
+__all__ = ["flag_outliers"]
+
+
+def flag_outliers(
+    self, sigma = 3
+):
+    """
+    Flag outliers on a chromatic rainbow.
+
+    Parameters
+    ----------
+    sigma : integer
+        Standard deviations (sigmas) allowed for individual data points before they are flagged as outliers.
+
+    Returns
+    -------
+    rainbow : Rainbow
+        A new Rainbow object with the outliers flagged as 0's in the 'ok' array.
+    """
+
+    
+    # create a history entry for this action (before other variables are defined)
+    h = self._create_history_entry("flag_outliers", locals())
+    
+    # create a copy of the existing rainbow
+    new = self._create_copy()
+    
+    list_of_rows = []
+    for i, row in enumerate(new.flux):
+        diffs = np.diff(row)
```

Comment:
I'm glad to see this works pretty well just with `diff`. In the future I may take a look at syncing this up with `remove_trends` to give slightly more nuanced control over how we ignore astrophysical signals, but if this works for now let's keep it as is!